src/main.cpp

```diff
@@ -3004,6 +3004,22 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, CBlockIn
         }
     }
 
+    // Coinbase transaction must include founders' reward if the
+    // block height is below 210000, which is the first halvening.
+    // The genesis block is excused from this requirement.
+    if ((nHeight > 0) && (nHeight < FOUNDERS_REWARD_UNTIL_BLOCK)) {
```

Comment:
This should probably be ``&& (nHeight <= FOUNDERS_REWARD_UNTIL_BLOCK)``, since that makes the total reward to the founders equal to ``FOUNDERS_REWARD_UNTIL_BLOCK * FOUNDERS_REWARD_AMOUNT_PER_BLOCK`` (and perhaps it should be renamed to ``FOUNDERS_REWARD_BLOCKS``).